.github/workflows/benchmarking.yml

```diff
@@ -96,7 +104,7 @@ jobs:
         # since it might generate duplicates when calling ingest_perf_test_result.py
         rm -rf perf-report-staging
         mkdir -p perf-report-staging
-        ./scripts/pytest test_runner/performance/ -v -m "remote_cluster" --skip-interfering-proc-check --out-dir perf-report-staging --timeout 3600
+        ./scripts/pytest test_runner/performance/ -v -m "remote_cluster" --sparse-ordering --skip-interfering-proc-check --out-dir perf-report-staging --timeout 3600
```

Comment:
Does `--sparse-ordering` also imply "run tests in order of definition"? I don't see any `@pytest.mark.order` calls.

If so, let's add a comment by the test definitions that the order is important.